README.md

```diff
@@ -1,340 +1,535 @@
 
 ```yaml
-privKey: /data/operator1/encrypted_private_key.json
-password: /data/operator1/password
-port: 3030
-storeShare: false
-logLevel: info
-logFormat: json
-logLevelFormat: capitalColor
-logFilePath: /data/output/operator1_logs_debug.log
+operatorIDs: [143, 219, 33, 34]    # array of Operator IDs which will be used for a DKG ceremony
+withdrawAddress: "0xa1a66cc5d309f19fb2fda2b7601b223053d0f7f4"    # Address where reward payments for the validator are sent
+owner: "0xb64923DA2c1A9907AdC63617d882D824033a091c"    # Address of owner of the Cluster that will manage the validator on ssv.network
+nonce: 0    # Owner nonce for the SSV contract
+network: "prater"    # Network name (default: mainnet)
+operatorsInfoPath: /data/operators_info.json    # Path to the file containing operators information
+# alternatively:
+# operatorsInfo: '{ 1: { publicKey: XXX, id: 1, ip: 10.0.0.1:3033 }'    # Raw content of the JSON file with operators information
+outputPath: /data/    # Path to store the output files
+initiatorPrivKey: /data/encrypted_private_key.json    # Path to private key of ssv initiator
+initiatorPrivKeyPassword: /data/password    # Path to password file to decrypt the key
+generateInitiatorKey: false # If set true - generates a new RSA key pair + random secure password. Result stored at `outputPath`
+logLevel: info    # Logger's log level (default: debug)
+logFormat: json    # Logger's encoding (default: json)
+logLevelFormat: capitalColor    # Logger's level format (default: capitalColor)
+logFilePath: /data/debug.log    # Path to file where logs should be written (default: ./data/debug.log)
 ```
 
-When using configuration file, run:
+> ℹ️ In the config file above, `/data/` represents the container's shared volume created by the docker command itself with the `-v` option.
+
+A special note goes to the `nonce` field, which represents how many validators the address identified in the owner parameter has already registered to the ssv.network.
+You can keep track of this counter yourself, or you can use the `ssv-scanner` tool made available by the SSV team to source it. For more information, please refer to the related user guide or to its [SDK documentation page](https://docs.ssv.network/developers/tools/ssv-scanner).
 
+> ℹ️ Note: For more details on `operatorsInfoPath` parameter, head over to the [Operators data section](#obtaining-operators-data) above
+
+Under the assumption that all the necessary files (`operators_info.json`, `encrypted_private_key.json`, `password`) are under the same folder (represented below with `<PATH_TO_FOLDER_WITH_CONFIG_FILES>`) you can run the tool using the command below:
 ```sh
-ssv-dkg start-operator --configPath "/examples/config/operator4.example.yaml"
+docker run --name ssv_dkg_initiator \
+-v "<PATH_TO_FOLDER_WITH_CONFIG_FILES>":/data -it \
+"ssv-dkg:latest" /app init --configPath /data/initiator.yaml && \
+docker rm ssv_dkg_initiator
```

Comment:
shouldn't the rm command be the first command? otherwise this will fail if the image already exist.
it works if this is the first time you run it, then it will delete the image and there will be no issue in the future.

Reply:
> it works if this is the first time you run it, then it will delete the image and there will be no issue in the future.

This is what I want to obtain. If the `rm` part is at the beginning, it will fail the first time and it won't "clean up after itself" when done.

This way, if people always use this command, they should never have an `ssv_dkg_initiator` container active.